app/navigation/AuthenticatedNavigator.scala

```diff
@@ -47,7 +47,7 @@ class AuthenticatedNavigator @Inject()() extends Navigator {
     case TaxYearSelectionPage => taxYearSelection(CheckMode)
     case AlreadyClaimingFREDifferentAmountsPage => alreadyClaimingFREDifferentAmount(CheckMode)
     case AlreadyClaimingFRESameAmountPage => alreadyClaimingFRESameAmount(CheckMode)
-    case YourAddressPage => yourAddress(CheckMode)
+    case YourAddressPage => _ => CheckYourAnswersController.onPageLoad()
```

Comment:
remove this line as unable to return to your address and navigator will return to CYA as default